### PR TITLE
Bug 1180399: Build fails if the default settings.xml is missing

### PR DESCRIPTION
**Pull request by jhadvig**
@bparees this is the fix I was mentioning.

### CODE REVIEW

File: cartridges/openshift-origin-cartridge-jbossas/bin/control [hunk 1]
```diff
@@ -240,8 +240,14 @@ function build() {
   then
       if force_clean_build_enabled_for_latest_deployment
       then
+          # Removing all Maven dependencies, except settings.xml
+          # NOTE: Redirecting to /dev/null casue the find cmd fails if the .m2/ dir is empty
```
Comment (bparees) [inline cartridges/openshift-origin-cartridge-jbossas/bin/control]: this note doesn't apply any more right?

**Comment by jhadvig** [comment]
[test][extended:cartridge]

**Comment by jhadvig** [comment]
[test][extended:cartridge]

**Comment by jhadvig** [comment]
[test][extended:cartridge]

**Comment by mfojtik** [comment]
LGTM, @bparees can you please have second look?

**Comment by bparees** [comment]
lgtm. [merge]

**Pull request merged**